[DOCS] Reorganize toctree for  Multilevel Video Understanding (#1699)
open-edge-platform/edge-ai-libraries: microservices/multilevel-video-understanding/docs/user-guide/get-started.md

   poetry lock --no-update
   poetry install
   ```

>    > **Note:** sometimes the `poetry install` may take long time, in this case, another option to install packages could be:

Reviewer: ```suggestion
   > **Note:** sometimes `poetry install` may take a long time; in that case, an alternative option to install packages could be:
```
